src/Messages.js

```diff
@@ -19,7 +19,7 @@ export default defineMessages({
     labelsApplicableSystemsCount: {
         id: 'labelsApplicableSystemsCount',
         description: 'applicable systems number label',
-        defaultMessage: '{systemsCount} applicable systems'
+        defaultMessage: '{systemsCount} affected systems'
```

Comment:
Just a nitpick, but this would be better as a new message. The title of it is  labelsApplicableSystemsCount but the message says affected systems. By this I mean keep this message here for keepsake and make a new message called labelsAffectedSystemsCount